### PR TITLE
Addition of Deployment Version 

### LINKED ISSUE #162: Hardcode release in a release.py
whatever you do, don't try some dynamic bs.

### PR DESCRIPTION
**Pull request by abaiken**
Closes #162

### CODE REVIEW

File: yupana/api/status/tests_model.py [hunk 1]
```diff
@@ -111,3 +111,10 @@ def test_startup_with_modules(self, mock_logger):  # pylint: disable=no-self-use
         """Test the startup method with a module list."""
         self.status_info.startup()
         mock_logger.assert_called_with(ANY, ANY, ANY)
+
+    def test_release_version_default(self):
+        """Test the release version default."""
+        release_version = self.status_info.release_version
+        git_commit = self.status_info.git_commit
+        expected = '0.0.0.%s' % git_commit
```
Comment (myersCody) [inline yupana/api/status/tests_model.py]: Should we use the `DEFAULT_VERSION` from `release.py `here just in case the default ever changes?

Comment (abaiken) [reply yupana/api/status/tests_model.py]: Yes, great idea 😄

**Review by myersCody — APPROVED** [review]

**Comment by kholdaway** [comment]
🤔 Are you done @abaiken ?

**Review by kholdaway — APPROVED** [review]
After 32 commits from Ashley, this looks great!

**Pull request merged**